spec/spec_helper.rb

```diff
@@ -38,7 +37,9 @@
 
 DaemonKit::Application.running!
 
+RSpec::Expectations.configuration.on_potential_false_positives = :nothing
```

Comment:
why this? ^^ which issue does this solve?

Reply:
but ^^'  surpress warnings whcih are potential failures if it's jsut noizy is not a good idea